.asf.yaml

```diff
@@ -21,16 +21,23 @@ github:
   collaborators:
     - krlmlr
     - nbenn
-    - ywc88
   enabled_merge_buttons:
     merge: false
     rebase: false
     squash: true
   features:
+    discussions: true
     issues: true
+    projects: true
```

Comment:
Do we want to use projects?

In apache/arrow, the MATLAB developers only use it: https://github.com/apache/arrow/projects?query=is%3Aopen

Reply:
Ah, okay. Let's keep them disabled for now.